src/v2/models/AppState.js

```diff
@@ -131,6 +131,7 @@ export default Model.extend({
     }
 
     this.set(resp);
+    this.set('__previousResponse', idx.rawIdxState);
```

Comment:
@nikhilvenkatraman-okta  I dont know what changes did you make but looks like we are stil settings things the old way and not clearing the old object we set. eg) AuthenticatorObject.

Reply:
yeah, I only included the change to use `__previousResponse `. Still testing the change to clear appState object before we set it. Should be a small change I'll include it in the next commit.